libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -236,6 +237,14 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  private listenForIonModalWillPresent() {
+    if (this.config.interactWithBackground && this.ionModalElement) {
+      this.ionModalElement.addEventListener('ionModalWillPresent', () => {
+        this.renderer.addClass(this.windowRef.document.body, 'allow-background-scroll');
```

Comment:
Consider extracting ` allow-background-scroll` to name variable (instead this magic string) and re-use when removing

Reply:
Moved string to private readonly property in next commit